docs/index.html

```diff
@@ -63,6 +63,16 @@ <h4>Accions amb texte</h4>
     <p> Per defecte, en les accions es mostren les icones corresponents, però si volem que el que es mostri és el texte haurem d'incloure al `hash` d'actions: <code>icons: false</code> </p>
```

Comment:
in Catalan?
We should document in English if we want this gem to be widely reused...

Reply:
At least we should start adding new content in English :D